lib/kvdb/kvdb_ctxn_internal.h

```diff
@@ -49,7 +49,7 @@ struct kvdb_ctxn_impl {
     struct kvdb_pfxlock      *ctxn_kvdb_pfxlock;
     struct kvdb_ctxn_pfxlock *ctxn_pfxlock_handle;
 
-    struct kvdb_ctxn_bind  *ctxn_bind;
+    struct kvdb_ctxn_bind   ctxn_bind;
```

Comment:
Move this down past ctxn_c0snr_set to keep the other field within the same cache line as much as possible...